packages/io/src/cor.ts

```diff
@@ -0,0 +1,26 @@
+import { CoR as ICoR, Middleware, Context, JsonValue } from './types'
+
+export class CoR implements ICoR {
+  private _middlewares: Middleware[] = []
+
+  public use(plugin: Middleware): void {
+    this._middlewares = [...this._middlewares, plugin]
```

Comment:
`this` could be returned so this method could be used as `cor.use().use().use()`